torch/csrc/autograd/profiler.cpp

```diff
@@ -437,6 +462,49 @@ static jit::CodeTemplate event_template(R"(
   "args": {}
 })");
 
+void writeProfilerEventsToStream(std::ostream& out, const std::vector<Event*>& events) {
+  TORCH_CHECK(out, "Could not open file");
+  Event* profiler_start = nullptr;
+  for (Event* e : events) {
+    if (0 == strcmp(e->name(), "__start_profile")) {
+      profiler_start = e;
+      break;
+    }
+  }
+  TORCH_CHECK(profiler_start, "Could not find __start_profile mark");
+
+struct PairHash {
```

Comment:
Hmm, must've missed this, will update. I can't run the autoformatter here (this file isn't formatted according to fbcode standards and formatting it will result in a bunch of irrelevant changes) and the internal formatter doesn't seem to complain.